packages/ibm-products/src/components/PageHeader/PageHeader.stories.js

```diff
@@ -338,6 +338,7 @@ const title = {
   },
   'Long title with icon': {
     text: 'A very long page title which will almost certainly have to be truncated at some point',
+    shortTitle: 'A shortened title for nav',
     loading: false,
     icon: (props) => <Bee size={24} {...props} />,
```

Comment:
Looks good. While this works, I think a separate variant like “With `shortTitle` in breadcrumbs” would be easier to capture this behavior.

If we want to reuse this one, I think “Long title with icon and shortTitle” might be better. Then to help with discoverability, we can change the title text? 🤔 

```suggestion
  'Long title with icon and shortTitle': {
    text: 'A very long page title with a short version in breadcrumbs; this will almost certainly be truncated at some point',
    shortTitle: 'A shortened title for nav',
    loading: false,
    icon: (props) => <Bee size={24} {...props} />,
```